bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
Is this going to be okay? Old possible values were `firefox` and `firefox-android`. New possible values would be `firefox`, `firefox-android`, `firefox-ios`, and `firefox-all`.

Didn't run in to any issues while manual/automated testing, but want to double check.